openwisp_controller/config/apps.py

```diff
@@ -120,6 +122,11 @@ def connect_signals(self):
             sender=self.config_model,
             dispatch_uid='devicegroup_templates_change_handler.backend_changed',
         )
+        post_save.connect(
+            self.org_allowed_devices.post_save_handler,
+            sender=self.org_model,
+            dispatch_uid='organization_allowed_devices_post_save_handler',
+        )
```

Comment:
What is this post save handler used for?

Reply:
It creates a OrganizationLimits object for every organization created.